src/include/platform/internal/GenericPlatformManagerImpl_FreeRTOS.ipp

```diff
@@ -275,6 +275,17 @@ void GenericPlatformManagerImpl_FreeRTOS<ImplClass>::PostEventFromISR(const Chip
 template <class ImplClass>
 void GenericPlatformManagerImpl_FreeRTOS<ImplClass>::_Shutdown(void)
 {
+    GenericPlatformManagerImpl<ImplClass>::_Shutdown();
+
+    if (mChipEventQueue != NULL) {
+        vQueueDelete(mChipEventQueue);
+        mChipEventQueue = NULL;
+    }
+
+    if (mChipStackLock != NULL) {
+        vSemaphoreDelete(mChipStackLock);
+        mChipStackLock = NULL;
+    }
```

Comment:
This affects/changes all the platforms using FreeRTOS?

Reply:
Yes, it does.  Assuming they call PlatformManager::Shutdown.

Reply:
Adjusted the PR to not delete things here.